### ISSUE TITLE
Paste image from clipboard

### ISSUE THREAD

**Issue by dandv** [olivier-m/minislate#7]
Is this possible, or within the scope of the project? I'm [comparing WYSIWYG editors](https://github.com/dandv/comparisons/blob/master/JavaScript-WYSIWYG-editors.md) and have been looking for this feature.

**Comment by basickarl** [comment]
Also been looking for this feature! @dandv Did you find any?

**Comment by dandv** [comment]
Yes, [Trumbowyg](https://github.com/dandv/comparisons/blob/master/JavaScript-WYSIWYG-editors.md#can-paste-images-and-rich-text).
